### ISSUE TITLE
add external audio

### ISSUE THREAD

**Issue by anvanvo** [MasayukiSuda/GPUVideo-android#110]
Is it possible to add external audio during recording?

**Comment by ToysoftInc** [comment]
Yes. I added bluetooth to my project.  You can modify the audio encoder and implement the bluetooth code there.
